mlxtend/classifier/tests/test_stacking_classifier.py

```diff
@@ -33,6 +33,7 @@
 y2 = np.c_[y, y]
 
 
+@pytest.mark.skip(reason="scikit-learn implemented a StackingClassifier in 0.22.")
```

Comment:
Yes, afaik it was based on mlxtend's StackingCVClassifier. Fun fact, when I was a student, I also helped implementing the VotingClassifier in scikit-learn based on the version here in mlxtend as well as the sequential feature selection class.